openexp/_color/color.py

```diff
@@ -71,23 +72,19 @@ def to_hex(colorspec):
 		arguments:
 			colorspec:
 				desc:	A color specification.
-				type:	[str, unicode, tuple, int]
+				type:	[str, unicode, array-like, int]
 
 		returns:
 			desc:	A hexadecimal color specification.
 			type:	unicode
 		"""
+		is_rgb = lambda c: hasattr(c, '__len__') \
+			and len(c) == 3 \
+			and all(isinstance(i, numbers.Number) and 0 <= i <= 255 for i in c)
```

Comment:
I hadn't considered that, but the specification is such that only integer values are allowed. So we check whether `i` is an instance of `numbers.Integral`.